cli/cli.py

```diff
@@ -523,6 +608,18 @@ def get_option_type_info() -> dict[str, Any]:
     return option_type_info
 
 
+def process_tables_options(opts: dict[str, Any], algo_name: str) -> dict[str, Any]:
+    if algo_name not in TASK_INFO[Task.ind].algos:
+        return opts
```

Comment:
There are going to be other algorithms that accept multiple tables. I was thinking of changing print_unused... so that it works with the actual kwargs and handles those options itself or something better, if possible.

Reply:
I think whoever is going to add another algorithm with this option would like being able to just add whatever is relevant to their algorithm into the globals and have things work, while having to understand as little as possible of the CLI.
On the contrary, of course, it may lead to problems if there are other options like it, but I believe it'd be better to make a general system that handles that case later and bear with the additional complexity right now.